docs/installation/index.md

```diff
@@ -1,33 +1,49 @@
 <!--[metadata]>
 +++
-title = "Supported installation"
+title = "Supported installations for Docker Engine"
 description = "Lists the installation methods"
 keywords = ["Docker install "]
 +++
 <![end-metadata]-->
 
-# Supported installation
+# Install Docker Engine 
 
-Docker supports installation on the following: 
+Docker Engine is supported on Linux, Cloud, Windows, and OS X. Installation instructions are available for the following: 
 
-* [Amazon EC2 Installation](amazon)
+## On Linux
 * [Arch Linux](archlinux)
-* [Microsoft Azure platform](azure)
-* [Installation from binaries](binaries)
 * [CentOS](centos)
 * [CRUX Linux](cruxlinux)
 * [Debian](debian)
 * [Fedora](fedora)
 * [FrugalWare](frugalware)
 * [Gentoo](gentoolinux)
-* [Google Cloud Platform](google)
-* [Install on Joyent Public Cloud](joyent)
-* [Mac OS X](mac)
 * [Oracle Linux](oracle)
-* [Rackspace Cloud](rackspace)
 * [Red Hat Enterprise Linux](rhel)
-* [IBM SoftLayer](softlayer)
 * [openSUSE and SUSE Linux Enterprise](SUSE)
 * [Ubuntu](ubuntulinux)
+
+## Other Linux
+If your linux distribution is not listed above, don't give up yet. To try out Docker on a distribution that is not listed above, go here: [Installation from binaries](binaries).
+
+## On Cloud
+* [Amazon EC2 Installation](amazon)
+* [Install on Joyent Public Cloud](joyent)
+* [Google Cloud Platform](google)
+* [IBM SoftLayer](softlayer)
+* [Microsoft Azure platform](azure)
+* [Rackspace Cloud](rackspace)
+
+## On OSX and Windows
+* [Mac OS X](mac)
 * [Windows](windows)
 
+## The Docker Archives 
+Instructions for installing prior releases of Docker can be found in the following docker archives:
+[Docker v1.7](http://docs.docker.com/v1.7/), [Docker v1.6](http://docs.docker.com/v1.6/), [Docker v1.5](http://docs.docker.com/v1.5/), and [Docker v1.4](http://docs.docker.com/v1.4/).
+
+## Where to go next
```

Comment:
Right ... and the best place for those steps would be at the end of each of the install pages.. I was using the call stack approach where most folks come back to the main page after doing the sub page for downloading..  How about something like... after installing on your selected platform be sure to check out these pages for next steps: